<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlertaInventario</class>
 <widget class="QMainWindow" name="AlertaInventario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>415</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Alertas Inventario</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/new/prefix2/icono.png</normaloff>:/new/prefix2/icono.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Alertas Inventario:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>401</width>
      <height>281</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/hicell.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>443</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuVenta">
    <property name="title">
     <string>Venta</string>
    </property>
    <addaction name="actionVenta"/>
    <addaction name="actionDetalle"/>
   </widget>
   <widget class="QMenu" name="menuInventario">
    <property name="title">
     <string>Inventario</string>
    </property>
    <addaction name="actionInventario"/>
    <addaction name="actionActualizar"/>
    <addaction name="actionAlertas"/>
   </widget>
   <widget class="QMenu" name="menuContabilidad">
    <property name="title">
     <string>Contabilidad</string>
    </property>
    <addaction name="actionContabilidad"/>
    <addaction name="actionEgresos"/>
   </widget>
   <widget class="QMenu" name="menuConfiguraci_n">
    <property name="title">
     <string>Configuración</string>
    </property>
    <addaction name="actionCambio"/>
    <addaction name="actionActualizar_Usuario"/>
   </widget>
   <addaction name="menuVenta"/>
   <addaction name="menuInventario"/>
   <addaction name="menuContabilidad"/>
   <addaction name="menuConfiguraci_n"/>
  </widget>
  <action name="actionVenta">
   <property name="text">
    <string>Venta</string>
   </property>
  </action>
  <action name="actionDetalle">
   <property name="text">
    <string>Modificar Venta</string>
   </property>
  </action>
  <action name="actionInventario">
   <property name="text">
    <string>Inventario</string>
   </property>
  </action>
  <action name="actionActualizar">
   <property name="text">
    <string>Actualizar</string>
   </property>
  </action>
  <action name="actionAlertas">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Alertas</string>
   </property>
  </action>
  <action name="actionContabilidad">
   <property name="text">
    <string>Contabilidad</string>
   </property>
  </action>
  <action name="actionEgresos">
   <property name="text">
    <string>Egresos</string>
   </property>
  </action>
  <action name="actionCambio">
   <property name="text">
    <string>Cambio de Contraseña</string>
   </property>
  </action>
  <action name="actionActualizar_Usuario">
   <property name="text">
    <string>Actualizar Usuario</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
